### PR TITLE
Create E2E Pegasus Pipeline

### PR DESCRIPTION
**Pull request by joecummings**
Big PR creating a Pegasus Pipelilne

This pegasus pipeline does the following:
```
preprocessing ------ claim_detection ------- srl_processing ------wikidata ------ unify
                                \______________ amr_parsing ________/ 
```

Currently, the output from unify looks like:
```
{
   "text": str,
   "template": str,
   "doc": str,
   "qnodes": List of interesting qnodes (Claimers and args)
}
```

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+from distutils.core import setup
+
+from setuptools import find_packages
+
+
+setup(
+    name="cdse_covid",
+    packages=find_packages(),
+    # 3.6 and up, but not Python 4
+    python_requires=">=3.7",
```
[the remark this answers is not in the document]
Comment (joecummings) [reply setup.py:12]: Comment is old, but you are correct.

File: setup.sh [hunk 2]
```diff
@@ -32,6 +32,9 @@ fi
 conda activate transition-amr-parser
 set -u  # /hack
 
+# Install proper SpaCy model
+python -m spacy download en_core_web_sm
```
[the remark this answers is not in the document]
Comment (joecummings) [reply setup.sh:36]: Hmm, that is true. I will look into why that is necessary.

File: wikidata_linker/wikidata_linking.py [hunk 3]
```diff
@@ -134,17 +135,16 @@ def get_request_kgtk(
...
-        return candidates
+    candidates = list(kgtk_response.json())
+    for candidate in candidates:
+        if not candidate["label"]:
+            candidate["label"] = [""]
+    if filter_results:
+        candidates = list(filter(make_kgtk_candidates_filter(query), candidates))
+    with open(cache_file, "w") as file:
+        json.dump(candidates, file, ensure_ascii=False, indent=2)
+        f.write("\n")
```
Comment (elizlee) [inline wikidata_linker/wikidata_linking.py:146]: Need to change `f` --> `file`

File: cdse_covid/pegasus_pipeline/claim_pipeline.py [hunk 4]
```diff
@@ -0,0 +1,154 @@
...
+        --output {amr_output_dir} \
+        --spacy-model {model_path} \
+        --amr-parser-model {amr_params.existing_directory("model_path")}
+        """,
+        override_conda_config=CondaConfiguration(
+            conda_base_path=params.existing_directory("conda_base_path"),
+            conda_environment="transition-amr-parser"
+        ),
+        depends_on=[claim_detection_output]
+    )
```
Comment (elizlee) [inline cdse_covid/pegasus_pipeline/claim_pipeline.py:91]: Add 8G resource_request

**Review by elizlee — CHANGES_REQUESTED** [review]
Just a couple small changes, the rest looks good!

File: requirements.txt [hunk 5]
```diff
@@ -215,6 +215,7 @@ types-six==0.1.7
 types-toml==0.1.3
 typing-extensions==3.10.0.0
 urllib3==1.26.6
+vista_pegasus_wrapper
```
Comment (elizlee) [inline requirements.txt:218]: The package name is `pegasus-wrapper`

**Pull request merged**